### PR TITLE
Improvement in ActiveRecord::Persistence doc [ci skip]

### PR DESCRIPTION
**Pull request by sonalkr132**

### CODE REVIEW

File: activerecord/lib/active_record/persistence.rb [hunk 1]
```diff
@@ -102,21 +102,21 @@ def persisted?
...
     #
-    # By default, save always run validations. If any of them fail the action
+    # By default, +save+ always runs validations. If any of them fail the action
     # is cancelled and +save+ returns +false+. However, if you supply
     # validate: false, validations are bypassed altogether. See
     # ActiveRecord::Validations for more information.
     #
-    # By default, #save also sets the +updated_at+/+updated_on+ attributes to
+    # By default, +#save+ also sets the +updated_at+/+updated_on+ attributes to
     # the current time. However, if you supply <tt>touch: false</tt>, these
```
Comment (sonalkr132) [inline activerecord/lib/active_record/persistence.rb]: @claudiob You had changed `+save+` to `#save` in bb78af73. However, at many other places method names are used as `+method_name+` and even `+#method_name+`.

Is there a preferred syntax?

Comment (sgrif) [inline activerecord/lib/active_record/persistence.rb]: If it is a method name, it should not be wrapped in `+`

**Comment by sonalkr132** [comment]
Thanks @sgrif. I have updated my PR.

Another quick question: Is changing [this if !condition](https://github.com/rails/rails/blob/master/activerecord/lib/active_record/persistence.rb#L231) to `unless` worth a PR?

**Comment by sgrif** [comment]
No. We don't accept cosmetic changes

**Comment by sonalkr132** [comment]
Gotya!

File: activerecord/lib/active_record/persistence.rb [hunk 2]
```diff
@@ -292,7 +293,7 @@ def update_column(name, value)
     #   user.update_columns(last_request_at: Time.current)
     #
     # This is the fastest way to update attributes because it goes straight to
-    # the database, but take into account that in consequence the regular update
+    # the database, but takes into account that in consequence the regular update
```
Comment (senny) [inline activerecord/lib/active_record/persistence.rb]: I think `take` was right here. As I read it, it refers to the reader to "take something into account".

**Comment by senny** [comment]
I'm not sure we should merge this. Most of the changes seem cosmetic. Linking the methods in rDoc does make sense but it's currently written in our [API doc guidelines](http://guides.rubyonrails.org/api_documentation_guidelines.html#fixed-width-font) to use fixed with fonts for method names. This is something we are changing for Rails 5 and I am working on a full overhaul of the Active Record docs to make use of rDocs linking. To prevent tons of merge conflicts I'd rather change this in one go though than through a series of tiny PR's.

**Comment by sonalkr132** [comment]
@senny  Thank you for review.
I was a bit confused about which convention is follow. I asked about it [here](https://github.com/rails/rails/pull/21791#discussion_r40508081). I have tried to reduce the noise :stuck_out_tongue: Please have a look.

> I am working on a full overhaul of the Active Record docs to make use of rDocs linking.

I would be more than happy to lend a hand. I have only started with reading the code base, so probably I will be reading a lot of comments anyway.

**Comment by senny** [comment]
@sonalkr132 I have a local branch that already changes most of Active Record. I'll try to move that forward so I can bring it into `master`. Once that's done you're welcome to join in :yellow_heart:

**Comment by sonalkr132** [comment]
@senny Thank you!

**Comment by sonalkr132** [comment]
@senny Hey thoughts on this getting merged?

**Comment by arthurnn** [comment]
Thanks @sonalkr132 .

@sgrif @senny I went ahead and merged this. I think the doc changes in here look good to go, and if you have more feedback related we can probably iterate over that version

**Pull request merged**